src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo__Img"
+        />
+      </a>
+
+      <nav>
+        <ul class="ul__section">
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line between the closing `</a>` tag and the `<nav>` tag to improve readability.